code/game/mecha/mecha_topic.dm

```diff
@@ -158,6 +158,15 @@
 	output += "</body></html>"
 	return output
 
+/obj/mecha/proc/get_log_tgui()
+	var/list/data = list()
+	for(var/list/entry in log)
+		data.Add(list(list(
+			"time" = time2text(entry["time"], "DDD MMM DD hh:mm:ss"),
+			"year" = GLOB.game_year,
```

Comment:
Simplify these timestamps. We dont care about game year or date, just the time itself.